src/graphql/subscription/dao-active.subscription.gql

```diff
@@ -42,10 +59,116 @@ subscription activeDaoSubs($regexp: String!) {
         price: details_planPrice_a
         isInfinite: details_isInfinite_i
         discountPercX10000: details_discountPercX10000_i
+
+        pricingplan {
+          maxMemberCount: details_maxMemberCount_i
+        }
       }
     }
 
+    # multisigs: openmultisig {
```

Comment:
@arsenijesavic why is this commented out? Should be deleted ( if not used ) or marked with TODO / FIXME comment if needed

Reply:
@arsenijesavic what about this?